subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolvedConfigurationBuildOperationIntegrationTest.groovy

```diff
@@ -169,4 +160,74 @@ class ResolvedConfigurationBuildOperationIntegrationTest extends AbstractHttpDep
         op.result.resolvedDependenciesCount == 1
     }
 
+    def "script classpath dependencies are exposed"() {
```

Comment:
We should test init scripts, settings scripts, and script plugins here.

Reply:
added test coverage for that